tmt/cli.py

```diff
@@ -569,14 +569,30 @@ def export(context, format_, nitrate, bugzilla, **kwargs):
     if bugzilla and not nitrate:
         raise tmt.utils.GeneralError(
             "The --bugzilla option is supported only with --nitrate for now.")
-    for test in context.obj.tree.tests():
-        if nitrate:
+
+    if nitrate:
+        for test in context.obj.tree.tests():
             test.export(format_='nitrate')
+    elif format_ == 'execute':
```

Comment:
The `execute` format is not needed from the command line. It is just used by the `detach` executor (which btw will be removed soon).

Reply:
Good thanks.